index.html

```diff
@@ -5021,55 +5021,28 @@ <h3 id="playback-AudioBufferSourceNode">
             The description of the algorithm is as follows:
           </p>
           <pre>
-// The AudioBuffer and context employed by this node
-var buffer;
-var context;
+let buffer;  /* AudioBuffer employed by this node */;
+let context; /* AudioContext employed by this node */;
```

Comment:
Sure, this went through a couple of iterations and got messed up in the process.